app/scripts/modules/core/src/pipeline/executions/executionGroup/ExecutionGroup.tsx

```diff
@@ -55,6 +55,8 @@ export class ExecutionGroup extends React.Component<IExecutionGroupProps, IExecu
     const pipelineConfig = find(this.props.application.pipelineConfigs.data, {
       name: this.props.group.heading,
     }) as IPipeline;
+
+    const hasNonMPTV2PipelineConfig = pipelineConfig && pipelineConfig.schema !== 'v2';
```

Comment:
since there are three different places we're checking `config.schema` against "v2", maybe abstract into some sort of helper function or at least throw v2 into a constant?

Reply:
if it's easier to keep the way it is i don't really have a strong opinion about this

Reply:
Great point, i'll dry this up

Reply:
updated, https://github.com/spinnaker/deck/pull/6644/commits/292cb5b53f1e600fadb819e704058b896e63c0b3